source/OutgoingMessages.Interfaces/Models/MeteredDataForMeteringPoint/ForwardMeasurementsMessageSeriesDto.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace Energinet.DataHub.EDI.OutgoingMessages.Interfaces.Models.MeteredDataForMeteringPoint;
 
-public sealed record ForwardMeteredDataMessageSeriesDto(
+public sealed record ForwardMeasurementsMessageSeriesDto(
```

Comment:
```suggestion
public sealed record SendMeasurementsMessageSeriesDto(
```

The correct name for the BRS is (now) `SendMeasurements` AFAIK.

![image](https://github.com/user-attachments/assets/fe2e2584-5013-499f-9351-9361096e0f2f)